tasks/src/document-question-answering/about.md

```diff
@@ -0,0 +1,45 @@
+## Use Cases
+
+Document question answering models can be used to answer natural language questions directly from document images. This is useful when the visual information of the document is important for answering the question.
```

Comment:
not sure we want to mention this here -- but the models are also able to answer questions about documents _without_ images. E.g. the LayoutLM variants can take text + bounding boxes as input, so word documents, excel spreadsheets, pdfs with text, etc. can be processed w/ their inherent structure and w/out having to create an image.

Reply:
I think we can have two use cases one including from PDFs another from text.